psa_car_controller/psacc/application/charge_control.py

```diff
@@ -94,6 +103,11 @@ def process(self):
                             thread = threading.Timer(periodicity, self.process)
                             thread.setDaemon(True)
                             thread.start()
+            elif status == STOPPED and has_threshold and hit_threshold and self.__is_approaching_scheduled_time(now):
+                logger.info("Approaching scheduled charging time, but should not charge. Postponing charge hour!")
+                self.force_update(vehicle_status)
```

Comment:
Why do you need to call this method ?

Reply:
I saw that it was also called before `self.control_charge_with_ack(False)`, so I figured I should do the same just before postponing the scheduled charge time. Not sure it's needed, but maybe it helps ensuring the vehicle is awake so that it receives the command?